.gitattributes

```diff
@@ -1,29 +1,6 @@
 # Auto detect text files and perform LF normalization
 * text=auto
 
-# Custom for Visual Studio
-*.cs     diff=csharp
-*.sln    merge=union
-*.csproj merge=union
-*.vbproj merge=union
-*.fsproj merge=union
-*.dbproj merge=union
-
-# Standard to msysgit
-*.doc	 diff=astextplain
-*.DOC	 diff=astextplain
-*.docx diff=astextplain
-*.DOCX diff=astextplain
-*.dot  diff=astextplain
-*.DOT  diff=astextplain
-*.pdf  diff=astextplain
-*.PDF	 diff=astextplain
-*.rtf	 diff=astextplain
-*.RTF	 diff=astextplain
-
 # Release export attributes
 technic/spec/** export-ignore
-.github/** export-ignore
-.luacheckrc export-ignore
-.mailmap export-ignore
-integration-test.sh export-ignore
+.* export-ignore
```

Comment:
This could as well be ``**/.* export-ignore` to make it match in all subdirectories too.
There's probably no dot files that should be exported / archived.

Reply:
```
technic$ find . -name '.*'
.
./.textures
./technic/spec/fixtures/mineunit/.git
./.gitmodules
./.gitattributes
./.github
./.mailmap
./.gitignore
./.git
./.luacheckrc